packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -266,4 +266,17 @@ gulp.task('icons-lg', ['icons-common'], () => {
   return merge(react, datauri, rawDatauri);
 });
 
-gulp.task('default', ['elements', 'spinners', 'icons-sm', 'icons-lg']);
+gulp.task('icons-font', ['icons-common'], () => gulp
+  .src('src/icons/**/*.svg')
+  .pipe(chmod(0o644))
+  .pipe(iconfont({
+    fontName: 'BpkIcon', // required
+    prependUnicode: false,
+    formats: ['ttf', 'eot', 'woff'], // default, 'woff2' and 'svg' are available
+    normalize: true,
+    fontHeight: 1001,
```

Comment:
Might be good to pop a comment here about why we're using `1001`, for any future contributors.

Reply:
Yeah I agree - I have no idea what this is doing.

Reply:
good shout, will do